### DISCUSSION TITLE
选择debugger的机制是怎么样的？

### DISCUSSION THREAD

**Discussion by KawasumiMaii** [xmake-io/xmake#1758] [Q&A]
我用clang替换了gcc，但是那时没装lldb，这时默认用xmake r -d用的是gdb，然后我装了lldb，这时候调试器用的是lldb了，然后我换回了gcc，结果。。貌似还是用的lldb，我不知道是什么决定了使用哪个debugger，我有地方可以配置吗？

**Comment by waruqi** [comment]
xmake f --debugger=gdb

具体看文档和 --help 都有说明
